.gitignore

```diff
@@ -0,0 +1 @@
+test.md
```

Comment:
Please add `node_modules` in the .gitignore file because it's a requirement in this project.

Reply:
Hello @gemmen29, Thanks for the review. Altho, are the [.hintrc](https://github.com/microverseinc/linters-config/blob/master/html-css/.hintrc) and [.stylelintrc.json](https://github.com/microverseinc/linters-config/blob/master/html-css/.stylelintrc.json) files suppose to be in the remote repo or its enough if those are in local machine.